src/darwin/Framework/CHIP.xcodeproj/project.pbxproj

```diff
@@ -626,7 +638,6 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
```

Comment:
Why this change?

Reply:
It's unnecessarily defined as "nothing" in the pbxproj file, just took it out because it's not needed for tests.